crates/test/mock-consensus/src/builder.rs

```diff
@@ -5,12 +5,17 @@
 /// Most importantly, defines [`Builder::init_chain()`].
 mod init_chain;
 
-use {crate::TestNode, bytes::Bytes};
+use {
+    crate::TestNode, bytes::Bytes, penumbra_proto::penumbra::core::component::stake::v1 as pb,
+    rand_core::OsRng,
+};
 
 /// A buider, used to prepare and instantiate a new [`TestNode`].
 #[derive(Default)]
 pub struct Builder {
-    app_state: Option<Bytes>,
+    pub app_state: Option<Bytes>,
+    pub validators: Vec<pb::Validator>,
```

Comment:
todo: this should be a type that isn't `pb::Validator`. it also might melt into the extension trait, since this is feeding `with_penumbra_auto_app_state`.

in any case, another problem to sort out tomorrow!